hugo/data/path/manifest.json

```diff
@@ -0,0 +1,4 @@
+{
+  "maincss": "css/main-less.e98b440295af10263485.css",
```

Comment:
Does this **manifest.json** get auto-generated? Does it need to be checked into version control?